packages/ui-components/Common/Select/SelectScrollButton/index.tsx

```diff
@@ -0,0 +1,130 @@
+import { ChevronDownIcon, ChevronUpIcon } from '@heroicons/react/24/outline';
+import { useEffect, useRef, useState } from 'react';
+import { type FC, type RefObject } from 'react';
+
+import styles from '@node-core/ui-components/Common/Select/index.module.css';
```

Comment:
```suggestion
import styles from '@node-core/ui-components/Common/Select/SelectScrollButton/index.module.css';
```

Maybe use dedicate css-module to simplify code